spec/controllers/replies_controller_spec.rb

```diff
@@ -334,11 +340,12 @@
     it "sets reply_order correctly on the first reply" do
       reply_post = create(:post)
       login_as(reply_post.user)
+      reply_post.mark_read(reply_post.user)
```

Comment:
Why is this necessary?

Reply:
Because otherwise it goes 'OH NO YOU HAVE AN UNREAD REPLY' when you try to add one >.>

Reply:
We didn't need to do that before because the code to check for unread replies doesn't trip on the _post_. I guess I could change it to ignore the first reply too.